CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1
+
+### Vision
+- Changed behaviour on simultaneous `VisionManager` and `VisionReplayManager` instances creation to throwing an exception
+- Fixed `ArLane` in Replay mode
```

Comment:
Be more descriptive? Eg. `Fixed 'ArLane' not displayed in VisionReplayManager`?

Reply:
done

Reply:
I think the reason for incorrect display of `ARLane` was incorrect values of `ARCamera` which could also affect the users. I would focus on this change and maybe mention lane issues as the consequence. In my understanding `ARLane` values were correct.

Reply:
make sense